src/activities/Elsa.Activities.Sql/Activities/SqlCommand/ExecuteSqlCommand.cs

```diff
@@ -26,7 +26,7 @@ public class ExecuteSqlCommand : Activity
         [ActivityInput(
             UIHint = ActivityInputUIHints.Dropdown,
             Hint = "Allowed databases to run SQL.",
-            Options = new[] { "MSSQL Server", "PostgreSql" },
+            Options = new[] { "", "MSSQLServer", "PostgreSql" },
```

Comment:
This change will break existing users who are using this activity in production. I suggest we only add the empty string and leave the rest of the values unchanged (and perhaps open a separate PR where we can distinguish between technical name and display name if necessary).